ocfweb/docs/docs/staff/procedures/setting-up-lacp.md

```diff
@@ -0,0 +1,77 @@
+[[!meta title="Setting up bridging and link aggregation"]]
+
+Bridging and LACP will be configured through puppet, but these intructions should
+serve as background and to help debug.
+
+## /etc/network/interfaces
+
+On Debian we can configure the network interfaces to come up at boot by adding
+stanzas to `/etc/network/interfaces` or `/etc/network/interfaces.d/<iface>`.
+
+### Bonding
+
+We use bond-mode 4, or 802.3ad. Other types are available, like active-fallback or load balancing,
+but we use 802.3ad. Link aggregation needs to be configured on the switch as well to work.
+
+Configuring the bonding interface is relatively simple. One must first install `ifenslave`, and
+identify the physical interfaces that will be slaved to the bond virtual interface. Then, write
+the following to the config file:
+
+```
+auto bond0
+
+iface bond0 inet manual
+    slaves <iface> <iface>
+    bond-mode 4
+    bond-miimon 100
+```
+
+If the bond interface is going to be the primary interface on the host, you may want to change
+`manual` to `static` and add addressing information by adding `address`, `gateway`, and `netmask`
+fields to the stanza. If the bond interface is going to be part of a bridge, leave it as `manual`.
```

Comment:
If this also needs to be configured on the switch side, perhaps link back to it from this page.